pkg/controller/migplan/registry.go

```diff
@@ -70,10 +70,7 @@ func (r ReconcileMigPlan) ensureMigRegistries(plan *migapi.MigPlan) error {
 			Category: migapi.Required,
 			Message:  RegistriesEnsuredMessage,
 		})
-	} else {
-		plan.Status.DeleteCondition(RegistriesEnsured)
```

Comment:
@jortel is this not necessary for some reason?

Reply:
It's not necessary because of condition _staging_.